spec/unit/views/components/paginated_collection_spec.rb

```diff
@@ -53,7 +53,7 @@ def paginated_collection(*args)
 
     context "when specifying :param_name option" do
       let(:collection) do
-        posts = 10.times.map { Post.new }
+        posts = Array.new(10) { Post.new }
```

Comment:
This is also coming from a cop that we don't have explicitly enabled. I've used it and since it's just for a spec too, I don't mind at all having it on. I'll comment separately about the cops config.